components/browser/engine-gecko-beta/src/main/java/mozilla/components/browser/engine/gecko/GeckoEngine.kt

```diff
@@ -158,6 +160,11 @@ class GeckoEngine(
         this.webExtensionDelegate = webExtensionDelegate
 
         val tabsDelegate = object : WebExtensionController.TabDelegate {
+            // We use this map to:
+            // - Ensures a web extension can only close tabs it has opened
```

Comment:
Why is that the case? Looking at the tabs API documentation I can't see that restriction being mentioned?

Reply:
OK, follow-up filed: https://github.com/mozilla-mobile/android-components/issues/4965 (see comment below)

Reply:
OK, follow-up filed: https://github.com/mozilla-mobile/android-components/issues/4965 (see comment below)